Assets/3D/bevy_c3d.toml

```diff
@@ -0,0 +1,4 @@
+name = "bevy_c3d"
+description = "A .c3d asset loader plugin for the Bevy engine"
```

Comment:
```suggestion
description = "A .c3d motion capture asset loader plugin for the Bevy engine"
```

Reply:
Something to this effect is really helpful to explain what the c3d format is to people who've never encountered it before.

Reply:
I agree, your change is significantly more explanative